backend/hasher/hasher_test.go

```diff
@@ -0,0 +1,39 @@
+//go:build !plan9 && !js
+// +build !plan9,!js
+
+package hasher_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/rclone/rclone/backend/hasher"
+	"github.com/rclone/rclone/fstest"
+	"github.com/rclone/rclone/fstest/fstests"
+
+	_ "github.com/rclone/rclone/backend/all" // for integration tests
+)
+
+// TestIntegration runs integration tests against the remote
+func TestIntegration(t *testing.T) {
+	opt := fstests.Opt{
+		RemoteName: *fstest.RemoteName,
+		NilObject:  (*hasher.Object)(nil),
+		UnimplementableFsMethods: []string{
+			"OpenWriterAt",
+		},
+		UnimplementableObjectMethods: []string{
+			"MimeType",
```

Comment:
? I would have thought you could just pass this through?

Reply:
I hit import loop when I tried to pass it.